build/configs/rtl8730e/loadable_apps/defconfig

```diff
@@ -462,13 +468,19 @@ CONFIG_GPIO=y
 # CONFIG_LCD is not set
 CONFIG_BCH=y
 CONFIG_RTC=y
+# CONFIG_RTC_DATETIME is not set
+# CONFIG_RTC_HIRES is not set
 # CONFIG_RTC_ALARM is not set
 CONFIG_RTC_DRIVER=y
 # CONFIG_RTC_IOCTL is not set
-# CONFIG_WATCHDOG is not set
+
+CONFIG_WATCHDOG=y
+CONFIG_WATCHDOG_DEVPATH="/dev/watchdog0"
+# CONFIG_WATCHDOG_FOR_IRQ is not set
 # CONFIG_TIMER is not set
 CONFIG_MMINFO=y
 CONFIG_PRODCONFIG=y
+CONFIG_KLOGBUFFER=y
```

Comment:
Please update this file using menuconfig. Some configs should be not added.

Reply:
Updated all defconfigs based on menuconfig.
Have tried to align the selections, please take a look.